lib/Runtime/Debug/DiagObjectModel.cpp

```diff
@@ -2563,11 +2562,6 @@ namespace Js
                                         InsertItem(originalObject, object, propertyId, isConst, isUnscoped, &pMethodsGroupWalker);
                                     }
                                 }
-                                else if ((jsFunction->IsScriptFunction() && !jsFunction->GetFunctionProxy()->IsJsBuiltInCode()) || jsFunction->IsBoundFunction())
-                                {
-                                    // Adding special property length for the ScriptFunction, like it is done in JavascriptFunction::GetSpecialNonEnumerablePropertyName
```

Comment:
This function it references doesn't seem to exist anymore, unless my code search tools are blatantly lying to me.

Reply:
It is hard to remember - but I think only one I can find is ```JavascriptRegExpConstructor::GetSpecialNonEnumerablePropertyName```
so that means this may have been removed.

Reply:
Last commit LGTM.